scripts/bootstrap.sh.in

```diff
@@ -44,6 +46,16 @@ EOF
 
 exec > >(tee -ia "${LOGFILE}") 2>&1
 
+determine_os() {
+    # This way permit to determine All the recent OS like Centos, Ubuntu
+    # and Debian
+     if [ -f /etc/os-release ]; then
+         . /etc/os-release;
+         OS=$ID;
+         VER=$VERSION_ID;
```

Comment:
VER is a global variable?
If yes:
- use a longer name (`OS_VERSION` maybe)
- declare it like `OS`, at the top of the script.

Reply:
IIRC this has no effect yet, it's only exported in case we ever need to switch on versions, so we can safely remove it until we require it.